bitnami/influxdb/Chart.yaml

```
@@ -24,4 +24,4 @@ name: influxdb
 sources:
   - https://github.com/bitnami/bitnami-docker-influxdb
   - https://www.influxdata.com/products/influxdb-overview/
-version: 2.0.6
+version: 2.0.7
```

Since it is a new feature addition instead of a bug fixing, we should bump the minor version
```suggestion
version: 2.1.0
```

---

Incorporated comment